Source/WebKit/GPUProcess/GPUConnectionToWebProcess.cpp

```diff
@@ -340,7 +340,7 @@ void GPUConnectionToWebProcess::didClose(IPC::Connection& connection)
     // RemoteGraphicsContextsGL objects are unneeded after connection closes.
     m_remoteGraphicsContextGLMap.clear();
 #endif
-#if USE(GRAPHICS_LAYER_WC)
+#if USE(GRAPHICS_LAYER_WC) && ENABLE(WEBGL)
     remoteGraphicsContextGLStreamWorkQueue().dispatch([webProcessIdentifier = m_webProcessIdentifier] {
         WCContentBufferManager::singleton().removeAllContentBuffersForProcess(webProcessIdentifier);
     });
```

Comment:
This whole area feels incorrect.

Reply:
I think this is correct. WCContentBufferIdentifier is used only for WebGL now.
You can add ENABLE(WEBGL) for Source/WebKit/Shared/wc/WCContentBufferIdentifier.h too.
I don't know how to add media support GraphicLayerWC now. Anyway, it's a future plan. It's OK for now.